docs/tutorials/workflow-tutorial.md

```diff
@@ -45,9 +45,14 @@ For example, this line from our above example invokes the transaction `InsertGre
 await ctxt.invoke(Greetings).InsertGreeting(friend, noteContent);
 ```
 
-The syntax for invoking function `foo(args)` in class `Bar` is `ctxt.invoke(Bar).foo(args)`.
+The syntax for invoking function `fn(args)` in class `Cls` is `ctxt.invoke(Cls).fn(args)`.
 
-You can also invoke other workflows using the [ctxt.childWorkflow()](../api-reference/contexts#workflowctxtchildworkflow) method.
+You can also invoke other workflows with the [`ctxt.invokeChildWorkflow()`](../api-reference/contexts#workflowctxtinvokechildworkflow) method.
+The syntax for invoking child workflow `wf` in class `Cls` with argument `arg` is:
+
+```typescript
+const output = await ctxt.invokeChildWorkflow(Cls.wf, arg)
```

Comment:
Why is it not invokeChildWorkflow(Cls).wf(arg) like the other invokes?

Reply:
Discussed in https://github.com/dbos-inc/dbos-transact/pull/432